templates/instance/admin.rs.html

```diff
@@ -7,7 +7,7 @@
 
 @(ctx: BaseContext, instance: Instance, form: InstanceSettingsForm, errors: ValidationErrors)
 
-@:base(ctx, i18n!(ctx.1, "Administration of {0}"; instance.name.clone()).as_str(), {}, {}, {
+@:base(ctx, i18n!(ctx.1, "The administration of {0}"; instance.name.clone()).as_str(), {}, {}, {
```

Comment:
It sounds a bit weird to me, to have "the" in a page title, but maybe that's just me. :woman_shrugging: